oda_api/api.py

```diff
@@ -1361,16 +1361,19 @@ def __init__(self):
     def enabled(self):
         return self._callback is not None
 
-    def report_progress(self, stage: str=None, progress: int=50, substage: str=None, subprogress: int=None, message:str=None):
+    def report_progress(self, stage: str=None, progress: int=50, progress_max: int=100, substage: str=None,
+                        subprogress: int=None, message:str=None):
```

Comment:
Probably also `subprogress_max`?

Reply:
subprogress_max was added in the latest commit